_StoreMixin.js

```diff
@@ -223,6 +224,26 @@ define([
 			return result;
 		},
 
+		refreshCell: function (cell) {
+			var row = cell.row;
+			var self = this;
+
+			return this.collection.get(row.id).then(function (item) {
```

Comment:
On second thought, I'm realizing that it seems like it'd ordinarily be unlikely for it to make sense to get a `cell` object to pass to this method without a collection in place anyway, so maybe what I described is an exceedingly rare case.